### ISSUE TITLE
Add new Bitcoin RPC Method: generatetoaddress

### ISSUE THREAD

**Issue by msgilligan** [ConsensusJ/consensusj#58]
Running **OmniJ** RegTests against latest Omni Core / Bitcoin Core receives the following error:
```
    org.consensusj.jsonrpc.JsonRpcStatusException: The wallet generate rpc method is deprecated and will be fully removed in v0.19. To use generate in v0.18, restart bitcoind with -deprecatedrpc=generate.
    Clients should transition to using the node rpc method generatetoaddress
```

We can use the `deprecatedrpc=generate` workaround for now, but we need to add the new `generatetoaddress` RPC method to the Bitcoin JSON-RPC client and update the RegTest support (here and in OmniJ) to use the new RPC.

**Comment by msgilligan** [comment]
`generatetoaddress` was added with: https://github.com/ConsensusJ/consensusj/commit/31258090d0b3db62dfcd0db1431167a9fd1cf8d4 

PR #60 migrates the rest of the module away from using `generate`.

**Comment by msgilligan** [comment]
Fixed in PR #60.
